wordpress-mobile/WordPress-iOS
# Jetpack Section: add backup download notice

Part of #15191
Related PR: https://github.com/wordpress-mobile/WordPressKit-iOS/pull/344

<img src="https://user-images.githubusercontent.com/7040243/106963416-52755380-671f-11eb-86d4-1782e596587f.png" width="300">

### To test

1. Open Activity List or Backups screen
2. Trigger a backup
3. After the backup is finished you should see a notice at the top to download the file
4. Tap Download, you should be redirected to the browser with the option to download the file
5. Go back to the app, tap dismiss
6. Refresh the list, the notice should not appear again

(You can also dismiss on Calypso and check in the app, both platforms are synced)

PR submission checklist:
